resources/views/front/canonicals.blade.php

```diff
@@ -0,0 +1,22 @@
+@php
+    $path = request()->path() == '/' ? '' : request()->path();
+    $model = $model ?? $page;
+    $base_url = config('app.url');
```

Comment:
@BenCavens Ik gebruik momenteel de app url als base anders lost dit het probleem dat johnny had niet op.
Maar dit zorgt er wel voor dat we de locale voor de hreflang zelf moeten toevoegen wat jammer is.
Ik wist hier niet direct een betere oplossing voor. Misschien dat jij een idee hebt?

Reply:
Any reason waarom je kiest om dit in chief te voegen, en niet in tt repo zelf? 
Ik heb het nog niet zelf getest maar mogelijks kunnen we eens iets proberen via de locale package zelf. In de config daar kunnen we domains zetten voor de canonical urls. Pair uurtje!

Reply:
Ik had dit in chief gezet omdat dat in mijn ogen iets was dat bijna ieder project nodig zal hebben en het leek mij makelijker om de verschillende urls op te halen vanuit chief dan van buitenaf, maar als het anders kan kan dat ook natuurlijk.